pkg/kubelet/kubelet.go

```diff
@@ -836,7 +836,15 @@ func NewMainKubelet(kubeCfg *kubeletconfiginternal.KubeletConfiguration,
 		util.SetNodeOwnerFunc(klet.heartbeatClient, string(klet.nodeName)))
 
 	// setup node shutdown manager
-	shutdownManager, shutdownAdmitHandler := nodeshutdown.NewManager(klet.GetActivePods, killPodNow(klet.podWorkers, kubeDeps.Recorder), klet.syncNodeStatus, kubeCfg.ShutdownGracePeriod.Duration, kubeCfg.ShutdownGracePeriodCriticalPods.Duration)
+	klog.InfoS("Attempting to start node shutdown manager", "shutdownGracePeriodRequested", kubeCfg.ShutdownGracePeriod.Duration, "shutdownGracePeriodCriticalPods", kubeCfg.ShutdownGracePeriodCriticalPods.Duration, "podPriorityShutdownGracePeriods", kubeCfg.PodPriorityShutdownGracePeriods)
```

Comment:
the log message may cause confusion when as the module is only later started in `initializeRuntimeDependentModules`, minor nit that we update log text to "Creating node shutdown manager".  This is also clearer as the call should not fail on create (can only fail later on Start)